exercises/saddle-points/introduction.md

```diff
@@ -0,0 +1,7 @@
+# Introduction
+
+You work for a transportation company that needs to improve the efficiency of their routes.
+
+You have been tasked with developing a program that can help the company identify routes that are problematic.
+These are locations where there is no optimal solution based purely on distance.
+The routes through these areas can then be analyzed by experts in the company, to find better solutions.
```

Comment:
I don't have any idea here to be honest. We could crowd/forum-source it maybe?

Reply:
That's a good idea. I'll start a conversation there.